docs/DeveloperGuide.md

```diff
@@ -144,7 +144,7 @@ The `Model` component,
 
 The `Storage` component,
 * can save both address book data and user preference data in JSON format, and read them back into corresponding objects.
-* inherits from both `AddressBookStorage` and `UserPrefStorage`, which means it can be treated as either one (if only the functionality of only one is needed).
+* inherits from `AddressBookStorage`, `ApplicationsStorage`, `JobsStorage`,and `UserPrefStorage`, which means it can be treated as any combination of the four storage interfaces (if only the functionality of a subset is needed).
```

Comment:
Missing space before "and"; also "ApplicationsStorage" should be called "ApplicationStorage" instead